index.md

```diff
@@ -45,6 +45,7 @@ color via `NO_COLOR`.
 | [colored](https://github.com/mackwic/colored) | Rust crate for coloring terminal output | [1.7.0](https://github.com/mackwic/colored/blob/master/CHANGELOG.md#170-january-2019) |
 | [crayon](https://github.com/r-lib/crayon) | R package for colored terminal output | [2018-02-08](https://github.com/r-lib/crayon/commit/700800135d04408bf1c99426b3fec9a4073b8a97) |
 | [Dye](https://github.com/dduan/Dye) | Cross-platform terminal styling for Swift command-line interface | [0.0.1](https://github.com/dduan/Dye/releases/tag/0.0.1) |
+| [kleur](https://github.com/lukeed/kleur) | The fastest Node.js library for formatting terminal text with ANSI colors~! | [4.1.0](https://github.com/lukeed/kleur/releases/tag/v4.1.0) |
```

Comment:
I'd like to avoid such hyperbole in the list, can you change "The fastest" to just "A" and remove the exclamation mark?

Reply:
Okay, I just copied the description from the lib.